diff --git a/node_modules/xstreaming-player/dist/Component/Audio.js b/node_modules/xstreaming-player/dist/Component/Audio.js
index c8df1ae..a6a8e85 100644
--- a/node_modules/xstreaming-player/dist/Component/Audio.js
+++ b/node_modules/xstreaming-player/dist/Component/Audio.js
@@ -12,7 +12,7 @@ class AudioComponent {
             audioRender.id = this.getElementId();
             audioRender.srcObject = srcObject;
             // audioRender.play()
-            audioRender.autoplay = true;
+            // audioRender.autoplay = true;
             this._audioRender = audioRender;
             audioHolder.appendChild(audioRender);
         }
diff --git a/node_modules/xstreaming-player/dist/Component/Video.js b/node_modules/xstreaming-player/dist/Component/Video.js
index 9840482..da9806f 100644
--- a/node_modules/xstreaming-player/dist/Component/Video.js
+++ b/node_modules/xstreaming-player/dist/Component/Video.js
@@ -35,9 +35,9 @@ class VideoComponent {
             videoRender.muted = true;
             videoRender.playsInline = true;
             // videoHolder.style.aspectRatio = '16 / 9'
-            setInterval(() => {
-                videoRender.play();
-            }, 4);
+            // setInterval(() => {
+            //     videoRender.play();
+            // }, 4);
             videoRender.addEventListener('loadedmetadata', () => {
                 const videoWidth = videoRender.videoWidth;
                 const videoHeight = videoRender.videoHeight;
diff --git a/node_modules/xstreaming-player/dist/index.js b/node_modules/xstreaming-player/dist/index.js
index ddbff00..b0f7a77 100644
--- a/node_modules/xstreaming-player/dist/index.js
+++ b/node_modules/xstreaming-player/dist/index.js
@@ -405,8 +405,8 @@ class xStreamingPlayer {
                 this._webrtcChannelProcessors[name].destroy();
             }
             this._webrtcChannelProcessors = {};
-            this._inputDriver.stop();
-            this._keyboardDriver.stop();
+            this._inputDriver && this._inputDriver.stop();
+            this._keyboardDriver && this._keyboardDriver.stop();
         }
     }
     getIceCandidates() {
